app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -84,6 +84,22 @@ def post_analytics(result)
       analytics.multi_factor_auth_setup(**properties) if context == 'confirmation'
 
       analytics.track_mfa_submit_event(properties)
+
+      if UserSessionContext.authentication_context?(context)
+        irs_attempts_api_tracker.mfa_verify_phone_otp_submitted(
+          reauthentication: false,
+          success: properties[:success],
+        )
+      elsif UserSessionContext.reauthentication_context?(context)
+        irs_attempts_api_tracker.mfa_verify_phone_otp_submitted(
+          reauthentication: true,
+          success: properties[:success],
```

Comment:
I'd use `result.success?` here:
```suggestion
          success: result.success?,
```